app/models/response_snapshot.rb

```diff
@@ -0,0 +1,15 @@
+# Works same as Response.
+#
+# Uses the results stored in its ResponseFinished event instead of getting info realtime from R.
+class ResponseSnapshot < Response
```

Comment:
Goed punt. Enige waar ik mee zit is dat het lastig is om te testen dat de R-package niet wordt aangeroepen bij elk van de methods, maar zal daar een loopje voor maken in een test.